src/site/content/en/blog/responsiveness/index.md

```diff
@@ -14,7 +14,7 @@ tags:
 ---
 
 {% Aside 'important' %}
-We have chosen option 2 of [high quantile approximation](/responsiveness/#high-quantile-approximation) as the candidate for the new responsiveness metric, with [maximum event duration](/responsiveness/#measure-interaction-latency) as the definition for interaction latency. This metric will be available as `experimental.responsiveness` starting in the February 2022 CrUX BigQuery dataset, released on March 8, 2022.
+We have chosen option 2 of [high quantile approximation](/responsiveness/#high-quantile-approximation) as the candidate for the new responsiveness metric, with [maximum event duration](/responsiveness/#measure-interaction-latency) as the definition for interaction latency. This metric will be available as `experimental.responsiveness` starting in the February 2022 [CrUX BigQuery dataset](https://developers.google.com/web/tools/chrome-user-experience-report/bigquery/getting-started), released on March 8, 2022.
```

Comment:
Nit: can you make the link text just "CrUX BigQuery"?

Reply:
Yup, just pushed a commit to do that.